### ISSUE TITLE
Add the name of your project (or work) you want to contribute

### ISSUE THREAD

**Issue by BoboPlastic** [BoboPlastic/LandingPages#4]
Add title and short description of your project.

**Comment by mtahashahid** [comment]
Hey can you assign this to me?

**Comment by BoboPlastic** [comment]
> Hey can you assign this to me?

Yeah sure. You can contribute with works simply like any static webpage having beautiful designs of your choice, any landing page design and so on. Make sure that you must not change any existing contents.

**Comment by Kedarnath-Rothe** [comment]
Can I add drum kit website 
If yes, please assign this to me under hactoberfest
